StreamChatClient_v3/Config/ChatClientConfig.swift

```diff
@@ -29,7 +29,7 @@ public struct ChatClientConfig {
     
     public var channel = Channel()
     
-    public init(apiKey: String) {
+    public init(apiKey: APIKey) {
```

Comment:
We need a convenience init with String too

Reply:
Why?

Reply:
Public facing api should be as easy as possible. A close example is `BaseURL` in v2, we introduced a convenience init with `String` too so people don't have to write `.init("api_key")` for this param

Reply:
OK, let's postpone the convenience API discussion once we have the base API there.